test.py

```
@@ -0,0 +1,17 @@
+def greet(name):
```

Ruff

Missing return type annotation for public function `greet`

Add return type annotation: `None`

(ANN201)